.github/workflows/screenshots.yml

```diff
@@ -0,0 +1,34 @@
+name: Percy screenshots
+
+on:
+  workflow_call:
+  workflow_dispatch:
+
+concurrency:
+   group: screenshots-${{ github.head_ref || github.run_id }}
+   cancel-in-progress: true
+
```

Comment:
This change has snuck in to the 'move to separate workflow commit' – can we split it out into a separate commit?

Reply:
Ah which bit has snuck in, renaming the group?

Reply:
We added it in: https://github.com/alphagov/govuk-frontend/pull/2878/files#diff-1db27d93186e46d3b441ece35801b244db8ee144ff1405ca27a163bfe878957fR14

So just moved it over and made it unique

Reply:
Aah of course, the tests workflow already has concurrency set and this needs it applied as it's being split out.